extras/indexing/src/main/java/org/apache/rya/api/client/mongo/MongoInstall.java

```diff
@@ -147,22 +147,23 @@ private RyaDetails initializeRyaDetails(
             final InstallConfiguration installConfig) throws AlreadyInitializedException, RyaDetailsRepositoryException {
         final RyaDetailsRepository detailsRepo = new MongoRyaInstanceDetailsRepository(adminClient, instanceName);
 
-        // Build the PCJ Index details. [not supported in mongo]
-        final PCJIndexDetails.Builder pcjDetailsBuilder = PCJIndexDetails.builder().setEnabled(false);
+        if(installConfig.getFluoPcjAppName().isPresent()) {
+        	log.warn("Mongo does not have fluo support, use ignoring the configured fluo application name: " + installConfig.getFluoPcjAppName().get());
+        }
+        
+        // Build the PCJ Index details.
+        final PCJIndexDetails.Builder pcjDetailsBuilder = PCJIndexDetails.builder()
+                .setEnabled(installConfig.isPcjIndexEnabled());
 
         final RyaDetails details = RyaDetails.builder()
                 // General Metadata
                 .setRyaInstanceName(instanceName).setRyaVersion(getVersion())
 
-                // FIXME RYA-215 .setGeoIndexDetails(new GeoIndexDetails(installConfig.isGeoIndexEnabled()))
-
                 // Secondary Index Values
+                // FIXME RYA-215 .setGeoIndexDetails(new GeoIndexDetails(installConfig.isGeoIndexEnabled()))
                 .setTemporalIndexDetails(new TemporalIndexDetails(installConfig.isTemporalIndexEnabled()))
                 .setFreeTextDetails(new FreeTextIndexDetails(installConfig.isFreeTextIndexEnabled()))//
-
-                // Entity centric indexing is not supported in Mongo DB.
-                .setEntityCentricIndexDetails(new EntityCentricIndexDetails(false))
-
+                .setEntityCentricIndexDetails(new EntityCentricIndexDetails(installConfig.isEntityCentrixIndexEnabled()))
```

Comment:
Did you enable entity centric indexing in this ticket too? Shouldn't that be a separate ticket?

Reply:
discussed in person